CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 1.4 RC 
+
+### Additions and Improvements 
+
+- New`trace_replayBlockTransactions` JSON-RPC API
+
+This can be enabled via the `--rpc-http-api TRACE` CLI flag.  There are some philosophical differences between Besu and other implementations that are outlined in the `[trace_rpc_apis.md](./docs/trace_rpc_apis.md)` documentation.
```

Comment:
Suggest "using" instead of "via". Also, what do you mean by "philosophical differences"? Perhaps just "differences" and leave it at that?

Reply:
There are two ways to view the correctness of the memory for a return call, as seen lower.  I wanted to qualify why we are different.

Reply:
Fair enough.